src/main/flight/gyroanalyse.h

```diff
@@ -24,9 +24,7 @@
 
 #include "common/filter.h"
 
-
-// max for F3 targets
-#define FFT_WINDOW_SIZE 32
+#define FFT_WINDOW_SIZE 32 // Max for F3
```

Comment:
F3 isn't that relevant any more, as it is no longer supported in Betaflight - should we consider increasing the window size?